tutorials/scripting/gdextension/gdextension_docs_generation.rst

```diff
@@ -0,0 +1,47 @@
+.. _doc_gdextension_docs_generation:
+
+Adding documentation to your GDExtension
+========================================
+
+.. note::
+    Adding documentation for GDExtensions is only possible for 4.3 and later.
+
+Similarly to how the engine works with built-in documentation inside the editor is how the
+generation of the GDExtension documentation works.
+
+Inside the Godot project directory of your GDExtension directory you run the following terminal command:
```

Comment:
> Godot project directory of your GDExtension directory

This may be 2am Max speaking, but I struggled a bit with this one. Can we make this more clear somehow? Maybe thats me.

Reply:
Will do a directory overview again like in the C++ GDExtension example 👍🏻 makes sense

Reply:
I agree. I am not sure which directory you mean. I've tried:

-  `./src/`
- `./godot-cpp/`
- `./project/addons/`
 
directories.

I only ever get an empty `doc_classes` directory in the project root.

**Shell output:**
```shell
godot --doctool ../ --gdextension-docs
Godot Engine v4.3.rc2.official.3978628c6 - https://godotengine.org

Loading docs...
Loading classes from: ../doc_classes
Merging docs...
Erasing old docs at: ../doc_classes
Generating new docs...
Deleting docs cache...
```

Reply:
@Structed Just saw your comment. You have to be inside the `project` directory in your case.

Reply:
That worked, thanks!
Perhaps that can be clarified in the docs?

Reply:
@Structed Great! Yeah I have a similar directory diagram in my local copy currently based on the one from basic GDExtension example page. That will hopefully clarify it.